XRTK-Core/Assets/XRTK/Inspectors/Profiles/MixedRealityControllerMappingProfilesInspector.cs

```diff
@@ -113,7 +113,7 @@ public override void OnInspectorGUI()
                 }
 
                 EditorGUILayout.BeginHorizontal();
-                profileChanged |= RenderProfile(controllerProfile, new GUIContent(profileName), false);
+                profileChanged |= RenderProfile(controllerProfile, new GUIContent(profileName), false, false);
```

Comment:
Does this mean the create new profile button is hidden for the mappings? With https://github.com/XRTK/XRTK-Core/pull/20 it's possible to create a proper mapping profile for each device.

Reply:
It's already hidden for the controller mappings. 

I think the issue was also resolved in #20